vm-setup/roles/v1aX_integration_test/tasks/move.yml

```diff
@@ -48,8 +48,13 @@
 
   # Install Ironic
   - name: Install Ironic
-    shell: "kustomize build /home/$USER/go/src/github.com/metal3-io/baremetal-operator/ironic-deployment/keepalived/ | kubectl --kubeconfig=/tmp/kubeconfig-{{ CLUSTER_NAME }}.yaml apply -n {{ NAMESPACE }} -f -"
-
+    shell: "{{ BMOPATH }}/tools/deploy.sh"
```

Comment:
sourcing will only work for the shell you're in at the moment, so you need to source here before calling deploy.sh, in the same shell

Reply:
Fixed